packages/fast-component-explorer/app/explorer.tsx

```diff
@@ -176,84 +186,106 @@ class Explorer extends Foundation<ExplorerHandledProps, {}, ExplorerState> {
             ),
             locationPathname,
             theme: ThemeName.light,
-            viewConfig: Object.assign({}, DesignSystemDefaults, {
-                direction: Direction.ltr,
-            }),
+            viewConfig: DesignSystemDefaults,
         };
     }
 
     public render(): React.ReactNode {
-        const explorerDesignSystem: DesignSystem = Object.assign(
-            {},
-            DesignSystemDefaults,
-            { density: -2 }
-        );
         return (
-            <DesignSystemProvider designSystem={explorerDesignSystem}>
-                <Background value={this.backgrounds.L1}>
-                    <Container className={get(this.props, "managedClasses.explorer")}>
-                        <Row style={{ flex: "1" }}>
-                            <Pane>
+            <Background value={neutralLayerL1}>
+                <Container className={get(this.props, "managedClasses.explorer")}>
+                    <Row style={{ flex: "1" }}>
```

Comment:
any specific reason for calling this inline instead of using a className as elsewhere?

Reply:
It was already there and hardcoded. If we need to add more styling to it later sure.